<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditBook</class>
 <widget class="QDialog" name="EditBook">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>429</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>170</y>
     <width>60</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string> ISBN：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>90</y>
     <width>45</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>书名：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>130</y>
     <width>60</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>作者：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="ISBNEdit">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>170</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>250</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>出版日期：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>81</x>
     <y>210</y>
     <width>60</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>出版社：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="RegNumEdit">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>50</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="PubHouseEdit">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>210</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="BookNameEdit">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>90</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="PubDateEdit">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>250</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>51</y>
     <width>60</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>登记号：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="AuthorEdit">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>130</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="SaveButton">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>310</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>保存</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
